spec/lib/airnow_api_adapter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe AirnowApiAdapter do
+  subject { AirnowApiAdapter.new }
+
+  describe '#readings_for' do
+    before do
+      stub_client = double()
```

Comment:
Useless assignment to variable - `stub_client`.<br>Do not use parentheses for method calls with no arguments.